cmd/tests.go

```diff
@@ -0,0 +1,43 @@
+package cmd
+
+import (
+    "github.com/spf13/afero"
+    "github.com/spf13/cobra"
+    testsDownload "github.com/supabase/cli/internal/tests/download"
+    testsPush "github.com/supabase/cli/internal/tests/push"
+    "github.com/supabase/cli/internal/utils/flags"
+)
+
+var (
+    testsCmd = &cobra.Command{
```

Comment:
We already have a `test` root command that we can enrich https://github.com/supabase/cli/blob/develop/cmd/test.go